terminator/lib/queue-handlers.js

```diff
@@ -150,24 +139,33 @@ export async function handleTransactionRetryQueueMessage(
       `Calculating gas fees from the recent Send message ${recentSendMessage.cid}`,
     )
 
+    const originalMaxPriorityFeePerGas = originalTx.maxPriorityFeePerGas
+    assert(
+      originalMaxPriorityFeePerGas !== undefined,
+      'originalTx.maxPriorityFeePerGas is null',
+    )
```

Comment:
According to TypeScript typings for viem, `originalTx.maxPriorityFeePerGas` may be undefined.

Do we know if that's actually possible and under which conditions it would happen? Do we need to handle this case or is it OK to keep my assert? What would happen when the assertion fails - will we drop the TX, keep retrying it forever, something else?

cc @pyropy